### DISCUSSION TITLE
Megane 3 2010 - press clutch to start the engine

### DISCUSSION THREAD

**Discussion by LemurMort** [cedricp/ddt4all#1271] [Q&A]
Hello all!
The car in question is Megane 3 1.6 16V, expression. I would like to start the car  with clutch pressed as now you only press the start button. I think it would be safer for the wife if it weren't that simple. Is it possible for this car to have this feature enabled? I've looked through the BCM and I couldn't find the feature.
Thanks!

**Comment by AndyIreland** [comment]
on our Clio mk4 2016 it was in our HFM module - i dont know if it will be same on yours:

![image](https://github.com/user-attachments/assets/d77c6913-474d-400e-8133-5c2366b98679)

**Reply by LemurMort** [reply]
Hello! I did find that field in BCM under "features activation logic". It does work but the message on the display says "press brake pedal". And it does work with brake pedal pressed also.
Any way to start the car with clutch only?

**Reply by AndyIreland** [reply]
I think the only time 'press clutch to start' came up in ours was when it was in gear

**Reply by LemurMort** [reply]
I'll check if this is in my case as well.
